### PR TITLE
test: upgrade node 18 and rename test to release

### PR DESCRIPTION
**Pull request by Dawntraoz**
- Upgrade node and semantic-release to v18
- Rename test.yml to release.yml

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -11,11 +11,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```
Comment (schabibi1) [inline .github/workflows/release.yml:14]: Quick questions: @Dawntraoz I was also configuring release.yml. I see that you set `actions/checkout@v3` and `actions/setup-node@v3`. Alba and @alexjoverm Should we align with v3 in all SDKs or what's the difference?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -11,11 +11,11 @@ jobs:
...
       - name: Setup Node
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
```
Comment (schabibi1) [inline .github/workflows/release.yml:16]: @Dawntraoz Same here

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -30,4 +30,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-        run: npx semantic-release
+        run: npx semantic-release@18
```
Comment (schabibi1) [inline .github/workflows/release.yml:33]: Also, do we need to specify `@18`? If so, should we align with SDKs?

**Comment by Dawntraoz** [comment]
@schabibi1 good points. I was configuring it, taking this one as a reference: https://github.com/storyblok/storyblok-nuxt/blob/main/.github/workflows/release.yml.

Maybe @roberto-butti can help give us some insights?

**Comment by Dawntraoz** [comment]
This will be covered in https://github.com/storyblok/storyblok-vue/pull/220

**Pull request closed without merging**